src/xia2/Modules/MultiCrystal/ScaleAndMerge.py

```diff
@@ -552,12 +562,14 @@ def run(self) -> None:
                 cluster_futures = {
                     pool.submit(
                         self._scale_and_report_cluster,
-                        self._data_manager,
+                        self._params,
+                        item.data_manager,
```

Comment:
Could we just pass the cluster objects as arguments to scale_and_report_cluster, rather than a long list of arguments that are the cluster properties?